### PR TITLE
Moar tests

### PR DESCRIPTION
**Pull request by dgageot**

### CODE REVIEW

File: pkg/skaffold/color/formatter_test.go [hunk 1]
```diff
@@ -67,6 +67,22 @@ func TestFprintf(t *testing.T) {
...
+func (n *nopCloser) Close() error { return nil }
+
+func TestFprintOnColoredWriter(t *testing.T) {
+	var b bytes.Buffer
+
+	coloredWriter := ColoredWriteCloser{
+		WriteCloser: &nopCloser{Writer: &b},
+	}
+
+	n, err := Green.Fprint(coloredWriter, "It's not easy being")
```
Comment (balopat) [inline pkg/skaffold/color/formatter_test.go]: being what? or just existing in general? :)

Comment (dgageot) [reply pkg/skaffold/color/formatter_test.go]: I think you have to ask @bobcatfish who wrote the original test!

Comment (bobcatfish) [reply pkg/skaffold/color/formatter_test.go]: bahahaha

Comment (bobcatfish) [reply pkg/skaffold/color/formatter_test.go]: image

**Pull request merged**